lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java

```diff
@@ -220,6 +226,7 @@ public BatchReport handleRequest(Request request, Context context) {
             forkJoinPool.shutdown();
         }
 
+        logReport(report);
```

Comment:
If you put this in the `finally` block, then you don't need to repeat it - and it's clearer what the intent is IMO

Reply:
I think that's a good shout - I'll shift it

Reply:
And yeah - just hashes of user IDs